{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "549342f6_1d8c5836",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1001064
      },
      "writtenOn": "2023-10-04T03:51:54Z",
      "side": 1,
      "message": "Fix the wrong python file name in the README.md",
      "revId": "34a7a72a85ba9cf867a14ad353d15fbb28562e7b",
      "serverId": "84e4e190-e623-4c37-9eda-58b1937c1b53"
    }
  ]
}